docs/UserGuide.md

```diff
@@ -103,18 +119,73 @@ Beyond individual products, you can also:
+
+![EditClientTommy](images/quick-start/EditClientTommy.png)
+
+4. We can also directly add these optional fields (email, address, etc.) when we are adding the client for the first time! So, `addClient n/Tammy Lim p/90127654 e/tammylim@hotmail.com bd/09091973` also adds in the client's email and birthday directly.
+
+<div markdown="span" class="alert alert-success">:exclamation: **Tips and tricks:**
+See [addClient](#311-adding-a-client-addclient) for more variety of optional fields!
+</div>
+
+5. Now we would like to add in an upcoming meeting with our client 'Bernice Yu' (index 2 in our sample data). We can do that by doing `addMeeting i/2 d/27102022 st/1400 et/1600 dn/Review plan` (You can replace `d/27102022` with today's date in the DDMMYYYY format)! When we hit enter, *MyInsuRec* will show us the list of meetings, including the one we just added. The image below shows *MyInsuRec* after adding the meeting with Bernice.
+
+![AddMeetingBernice](images/quick-start/AddMeetingBernice.png)
+
+6. To view the meeting (with index number of 1) in greater detail, we can call `viewMeeting i/1`. This brings up a more detailed view of the meeting, with information such as description and Bernice's contact number. The image below shows *MyInsuRec* after using the `viewMeeting` command.
+
+![ViewMeetingBernice](images/quick-start/ViewMeetingBernice.png)
+
+7. Now, to go back to the list of clients, simply type in `listClient`. This brings us back to the list of clients.
+
+8. We would like to define and add more products! We can do so via `addProduct pd/MyInsureCare`. This adds a product 'MyInsureCare' and also brings us to the list of product with the newly added product. The image below shows *MyInsuRec* adding the product.
+
+![AddProductMyInsureCare](images/quick-start/AddProductMyInsureCare.png)
+
+9. Now, we want to add a client named who has already purchased 'MyInsureCare'. To do that, we can once again type in `addClient n/Ng Jun Yi p/81230987 pd/MyInsureCare` to add a client and indicate that he has bought 'MyInsureCare'. The image below shows *MyInsuRec* after adding the client with the product.
+
+![AddClientJunYiWithProduct](images/quick-start/AddClientJunYiWithProduct.png)
+
+<div markdown="block" class="alert alert-warning">:exclamation: **Caution:**
+Only products added already via `addProduct` can be used! This is to help ensure the cleanliness of *MyInsuRec*. So, add your product via `addProduct` before using it to add a client with that product! See [addProduct](#331-adding-a-product-addproduct).
+</div>
+
+10. We want to delete the client 'Alex Yeoh' from our records! To do that, type in `delClient i/1`. The image below shows *MyInsuRec* after deleting the client.
```

Comment:
Here can also give reason as to why we want to delete the client.
Example, Alex Yeoh do not have any product with us anymore and would want his data to be protected etc...